addons/Assessments_Navigation_Bar/src/presenter.js

```diff
@@ -927,6 +960,37 @@ function AddonAssessments_Navigation_Bar_create(){
         }, this);
     };
 
+    presenter.NavigationManager.prototype.appendStaticPages = function (pages) {
+        presenter.$view.find('.navigation-buttons-first .section').remove();
+        presenter.$view.find('.navigation-buttons-last .section').remove();
+        var staticSections = [];
+        for (var i = pages.length - 1; i > -1; i--) {
+            var page = pages[i];
+            var sectionName = page.getSectionName();
+            var $section = null;
+            if (staticSections.indexOf(sectionName) == -1) {
+                $section = this.getSection(sectionName, page.getSectionClassName());
+                staticSections[sectionName] = $section;
+                if (page.staticPosition == 'left') {
+                    presenter.$view.find('.navigation-buttons-first .previous').after($section);
+                } else {
+                    presenter.$view.find('.navigation-buttons-last .next').before($section);
```

Comment:
Czy nie powinno być `after` zamiast `before` ? 
Jeżeli dobrze zrozumiałem funkcjonalność to spowoduje, że w sekcji z prawej strony będą dodawane strony statyczne w odwrotnej kolejności. 
Czyli taka konfiguracja:

> 1-2;;1,2;cover;left
> 3;;3
> 4;;4;cover;left
> 5;;5
> 6;;6
> 7;;7
> 8;;8
> 9;;9
> 10;;10
> 11;;11
> 12;;12
> 13;;13
> 14;;14
> 15;;15
> 16;;16
> 17;;17;grammar;right
> 18;;18;grammar;right

Spowoduje takie wyświetlenie: https://test-9053-dot-mauthor-dev.ew.r.appspot.com/present/4538519814930433
A przez to strzałka po prawej jest nieintuicyjna. Z 16 przejdzie na ostatni element czyli 17 a potem na 18.